src/components/users/List.vue

```diff
@@ -29,9 +29,7 @@ function parseRolesFromQuery(roles: string | string[] | undefined): UserRole[] {
 		return allRoles.filter((role) => role === roles)
 	}
 
-	const r: Set<UserRole> = new Set(allRoles.filter((r) => roles.includes(r)))
-
-	return Array.from(r.values())
+	return allRoles.filter((r) => roles.includes(r))
```

Comment:
Realised wrapping this in a Set is redundant. Construction guarantees no duplicates.